lms/templates/course_modes/track_selection.html

```diff
@@ -21,8 +21,7 @@
             $('button[name=verified_mode]').click(function(e){
                 e.preventDefault();
                 window.location.href = '${ecommerce_payment_page | n, js_escaped_string}?sku=' +
-                    encodeURIComponent('${sku | n, js_escaped_string}') +
```

Comment:
We need to revert this change. We need to keep appending the course_run_key identifier wherever we are